examples/http-routing/gateway.yaml

```diff
@@ -0,0 +1,15 @@
+kind: Gateway
+apiVersion: networking.x-k8s.io/v1alpha1
+metadata:
+  name: prod-web
+  namespace: default
```

Comment:
One tiny nit - I think we've traditionally avoided hardcoding a namespace in our examples. This makes it easier for users to deploy examples into specific test/demo namespaces if they want.